src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.service;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws AuthenticationException, RegistrationException;
+
+    User register(String email, String password) throws AuthenticationException;
```

Comment:
why method login throws RegistrationException? and the the register method does not? Maybe you need to change either logic or namings